static/app/views/replays/detail/network/index.tsx

```diff
@@ -234,6 +277,28 @@ function NetworkList({
             projectId={projectId}
             startTimestampMs={startTimestampMs}
           />
+          {sortConfig.by === 'startTimestamp' && showJumpUpButton() ? (
+            <Button
+              onClick={handleClick}
+              aria-label={t('Jump Up')}
+              priority="primary"
+              size="xs"
+              style={{position: 'absolute', justifySelf: 'center', top: '28px'}}
+            >
+              {t('Jump Up')}
```

Comment:
We should get some more input on this label.

"Jump up" is nice and short, but imo it's not descriptive enough when i see it in the UI. 

The rule is, if i'm going to comment i better have a suggestion... so how about "↑ Scroll to current time".
I like the up/down arrow (which slack uses), but 'current time' isn't the best either.

Reply:
what about something along the lines of "↑ Scroll to your timestamp"

Reply:
that's getting better yeah